x-pack/legacy/plugins/siem/server/lib/detection_engine/rules/update_rules.ts

```diff
@@ -90,8 +92,8 @@ export const updateRules = async ({
       tags: addTags(tags, rule.params.ruleId, rule.params.immutable),
       name,
       schedule: { interval },
-      actions: rule.actions,
-      throttle: rule.throttle,
+      actions: actions.map(transformRuleToAlertAction),
+      throttle: null,
```

Comment:
NOTE: I put in `throttle: null` just like the additional call above set it to by my previous note: https://github.com/elastic/kibana/pull/62904/files#r405279454

If this isn't suppose to ever be null for some reason let me know.